src/domains/chart/sagas.ts

```diff
@@ -134,7 +134,7 @@ function* fetchDataSaga({ payload }: Action<FetchDataPayload>) {
     before,
     dimensions,
     context: chart,
-    aggr_method: "avg",
+    aggr_method,
```

Comment:
We should keep `aggrMethod` attribute as camelCased on every place inside the code, and just pass it here as `underscored`
```
aggr_method: aggrMethod,
```